src/PixiEditor/ViewModels/Document/Nodes/Animable/TimeNodeViewModel.cs

```diff
@@ -0,0 +1,7 @@
+using PixiEditor.ChangeableDocument.Changeables.Graph.Nodes.Animable;
+using PixiEditor.ViewModels.Nodes;
+
+namespace PixiEditor.ViewModels.Document.Nodes.Animable;
+
+[NodeViewModel("TIME_NODE", "ANIMATION", "\uE900")]
```

Comment:
It's my personal preference, feel free to ignore it if you disagree, but `NodeViewModelAttribute` name could be `NodeSettings` or something, we already have `NodeViewModel<>` below, so we know it's a view model and settings feels more fitting.

Reply:
I think C# devs did the alias thing (being able to use attributes without the attribute append) just so you could give a class a attribute with the same name.
I think the name fits just as well as it is an info attribute about the view model